src/safeposix/syscalls/fs_calls.rs

```diff
@@ -2042,38 +2042,74 @@ impl Cage {
     }
 
     //------------------------------------IOCTL SYSCALL------------------------------------
-
+    //ioctl manipulates the underlying device parameters of special files. In particular, it is used as a way
+    //for user-space applications to interface with device drivers. The function accepts three parameters:
+    //fd - an open file descriptor that refers to a device, request - the control function to be performed
```

Comment:
For the request parameter, can you please give an example of what the `request` parameter looks like?